src/s3/commands/uploadFile.ts

```diff
@@ -418,15 +440,77 @@ export async function promptUserForBucket(
         }
     })
 
+    const lastTouchedFolder = globals.context.globalState.get<SavedFolder | undefined>('aws.lastTouchedS3Folder')
+    let lastFolderItem: BucketQuickPickItem | undefined = undefined
+    if (lastTouchedFolder) {
+        lastFolderItem = {
+            label: lastTouchedFolder.folder.name,
+            description: '(last opened S3 folder)',
+            bucket: { Name: lastTouchedFolder.bucket.name },
+            folder: lastTouchedFolder.folder,
+        }
+    }
+
+    const lastUploadedToFolder = globals.context.globalState.get<SavedFolder | undefined>('aws.lastUploadedToS3Folder')
+    let lastUploadedFolderItem: BucketQuickPickItem | undefined = undefined
+    if (lastUploadedToFolder) {
+        lastUploadedFolderItem = {
+            label: `Folder: ${lastUploadedToFolder.folder.name}`,
```

Comment:
I removed this "Folder:" prefix because 

1. inconsistent with `lastFolderItem` 
2. the `description` gives similar information